libr/bin/format/mz/mz.h

```diff
@@ -7,13 +7,15 @@
 #include "mz_specs.h"
 
 struct r_bin_mz_segment_t {
-	ut64 paddr;
-	ut64 size;
+	ut32 paddr;
+	ut32 vaddr;
+	ut32 size;
 	int last;
 };
 
 struct r_bin_mz_reloc_t {
-	ut64 paddr;
+	ut32 paddr;
```

Comment:
the idea to use ut64 in all those addresses is because the user can map an mz found in a really high address, because it can be found in a memory dump or a hard drive. it is really an issue?

Reply:
probably not, although I don't see how this would ever work on real DOS?

Reply:
> would be great if we can get rid of the use of btree_ in here, its the only place where this data structure is used and i want to move it into sdb/Attic for proper evaluation. We can use a hashtable instead

the reason this is used is to sort the segment list. so use an r_list and sort it with r_list_sort instead?